.github/workflows/ci_test-full.yml

```
@@ -27,7 +27,7 @@ jobs:
         exclude:
           # Skip if torch<1.8 and py3.9 on Linux: https://github.com/pytorch/pytorch/issues/50014
           - {os: ubuntu-20.04, python-version: "3.9", requires: "oldest"}
-        #include:
+        include:
           # nightly: add when there's a release candidate
           #- {os: ubuntu-20.04, python-version: "3.10", requires: "latest", release: "pre"}
```

This needs to be included too

```suggestion
          - {os: ubuntu-20.04, python-version: "3.10", requires: "latest", release: "pre"}
```

---

why? troch is not released in stable while it is only testing